python/ray/dataframe/__init__.py

```diff
@@ -2,6 +2,15 @@
 from __future__ import division
 from __future__ import print_function
 import threading
+import pandas as pd
+
+pd_version = pd.__version__
+pd_major = int(pd_version.split(".")[0])
+pd_minor = int(pd_version.split(".")[1])
+
+if pd_major != 0 or pd_minor < 22:
+    raise Exception("In order to use Pandas on Ray, please upgrade your Pandas"
+                    " version to >= 0.22.")
```

Comment:
this will only happen if we import `ray.dataframe` right? and not if we just do `import ray`

Reply:
Correct.